docsrc/imap/support/feedback-mailing-lists.rst

```diff
@@ -4,80 +4,29 @@
 Mailing Lists
 =============
 
+Cyrus is discussed at a few official mailing lists hosted by
+`Topicbox <https://cyrus.topicbox.com/>` — which is, itself, powered by Cyrus!
```

Comment:
This link is missing the magical `_` after the closing "`", which means instead of a link to topicbox, we get an inline code block containing "Topicbox <https://cyrus.topicbox.com/>" in italicised monospace 😅 (RST!)